patches/chrome-BUILD.gn.patch

```diff
@@ -60,31 +60,32 @@ index a8c87a5d595af314ff9592b5795346b6859ff60b..41ee7058879d3556d8feaaf2c632f0b0
        }
  
        extra_substitutions = [
-@@ -937,6 +944,7 @@ if (is_win) {
+@@ -935,7 +942,7 @@ if (is_win) {
+     if (enable_updater) {
+       if (is_chrome_branded) {
          sources += [ "//third_party/updater/chrome_mac_universal_prod/cipd/${updater_product_full_name}.app" ]
-       } else {
+-      } else {
++      } else if (brave_enable_updater && !build_omaha4) { sources += [ "//build/mac_files/omaha4/BraveUpdater.app" ] } else {
```

Comment:
`BraveUpdater` -> `${updater_product_full_name}`?

Reply:
Resolved in 16b2ab89a7e66d7c3dc45c884d26cd19ac877e2e.